src/main/java/StringCalculatorStep1.java

```diff
@@ -5,7 +5,8 @@ public static final int Add(final String numbers) {
         int returnValue = 0;
         String[] numbersArray = numbers.split(",");
         if (numbersArray.length > 2) {
```

Comment:
Splitting the 'numbers' string without trimming may lead to unexpected parsing errors. Trim the string before splitting.